internal/controllers/vehicle_subscription_controller.go

```diff
@@ -63,16 +73,57 @@ func (v *VehicleSubscriptionController) AssignVehicleToWebhook(c *fiber.Ctx) err
 		return c.Status(http.StatusBadRequest).JSON(fiber.Map{"error": "Invalid vehicle token ID format"})
 	}
 
+	// Validate vehicle permissions by calling the Identity API.
+	identityURL := fmt.Sprintf("https://api.dimo.org/identity/v1/vehicles/%s/sacds", vehicleTokenIDStr)
```

Comment:
is this the mobile api? I think this information is also on our own identity-api graphql endpoint?
Events/webhooks service will live in the DIMO storage node, as part of the protocol, so we'd want to avoid dependencies on DINC (DINC owns the mobile app and mobile backend) services.

Reply:
actually that url doesn't resolve... I know mobile backend is [https://api.dimo.co/](https://api.dimo.co/api#/compatible-integrations/CompatibleIntegrationsController_getCompatibleIntegrations)